docs/using-the-jdbc-driver/Telemetry.md

```diff
@@ -44,7 +44,30 @@ Before enabling the Telemetry feature, a few setup steps are required to ensure
 
 Telemetry for the AWS JDBC Driver is a monitoring strategy that overlooks all plugins enabled in [`wrapperPlugins`](../UsingTheJdbcDriver.md#connection-plugin-manager-parameters) and is not a plugin in itself. Therefore no changes are required in the `wrapperPlugins` parameter to enable Telemetry.
 
-In order to enable Telemetry in the AWS JDBC Driver all you need to do is to set the `enableTelemetry` property to `true` either in the code or in the connection string.
+In order to enable Telemetry in the AWS JDBC Driver you need to first set the `enableTelemetry` property to `true` either in the code or in the connection string.
+
+After that, it will be needed to, in the application code that will be using the driver, to set up the recorders that will take care of exporting the telemetry data from the code to the ADOT Collector.
```

Comment:
Can we convert this to bullet points?
1. Set the `enableTelemetry` property to `true`. You can either set it in the through `Properties` or through the connection string.
2. Set up the recorders that exports the telemetry data from the code to the ADOT Collector.

Reply:
How do you set up the recorders? If it is the sample code below the relation is not very obvious on first read. Can we merge them into the same paragraph?